src/spikeinterface/generation/drift_tools.py

```diff
@@ -125,13 +125,13 @@ def __init__(self, **kwargs):
 
     @classmethod
     def from_static(cls, templates):
```

Comment:
```suggestion
    def from_static_templates(cls, templates):
```
?

Reply:
I am OK with this.
@yger could you propagate this change at some places ?